Tests/InstallationTests/SPMCustomEntitlementComputationInstallation/InstallationTests.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  InstallationTests.swift
+//  InstallationTests
+//
+//  Created by Andrés Boedo on 10/28/20.
+//
+
+import XCTest
+
+import RevenueCat_CustomEntitlementComputation
+
+class InstallationTests: XCTestCase {
```

Comment:
I solved the testability issue by having new installation tests that use the new API through the new SPM 🎉